ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DBFetchingProcessRepository.scala

```diff
@@ -197,7 +191,7 @@ abstract class DBFetchingProcessRepository[F[_]: Monad](val dbConfig: DbConfig)
       modifiedBy = processVersion.user,
       createdAt = DateUtils.toLocalDateTime(process.createdAt),
       createdBy = process.createdBy,
-      json = processVersion.json.map(_ => convertToTargetShape(processVersion.graphProcess, process)),
+      json = processVersion.json.map(json => convertToTargetShape(GraphProcess(json), process)).getOrElse(Unit.asInstanceOf[PS]),
```

Comment:
Because of `BaseProcessVersionEntity` strategy..

Reply:
now it's rather unclear. Let's change signature:
`convertToTargetShape[PS: ProcessShapeFetchStrategy](graphProcess: Option[GraphProcess], process: ProcessEntityData)` => then it will become clear that `None` is only allowed for `NotFetch` strategy

Reply:
Good point @mproch.

Reply:
yep, now it looks more clear, thx